examples/webgl_loader_texture_exr.html

```diff
@@ -33,8 +33,17 @@
 
 			import { EXRLoader } from './jsm/loaders/EXRLoader.js';
 
+			const ToneMapping = {
+				Linear: THREE.LinearToneMapping,
+				Rienhard: THREE.ReinhardToneMapping,
+				RienhardLuma: THREE.ReinhardLumaToneMapping,
+				Cineon: THREE.CineonToneMapping,
+				ACES: THREE.ACESFilmicToneMapping,
+			};
+
 			const params = {
-				exposure: 2.0
+				logExposure: 0.0,
+				toneMapping: ToneMapping.Linear
```

Comment:
I do not think we add tone mapping options to this example. Save that for the tone mapping examples.